### ISSUE TITLE
Can work on iOS?

### ISSUE THREAD

**Issue by sameul2012** [HyNetwork/hysteria#337]
If yes, what's the best companion? 

How to do that? 

Thank you.

**Comment by mritd** [comment]
On IOS systems, you can use Shadowrocket or Stash to connect to the hysteria server.

**Comment by sameul2012** [comment]
No, on iOS , Shadowrocket cannot connect Hysteria, I tried many times.

Could you please help to indicate the specific instructions or steps?

You know, on Mac, I should open terminal, run Hysteria client and V2RayX.

But on iOS, there is no way to run Hysteria client.

**Comment by sameul2012** [comment]
> https://github.com/HyNetwork/hysteria/wiki/Installation#ios

Bro, that is only one link, to App Store...

**Comment by tobyxdd** [comment]
I can confirm that Stash & Shadowrocket both work.

> specific instructions or steps

tbf that's something you should ask around in the communities of these apps instead.
